src/main/java/nextstep/qna/domain/Answers.java

```diff
@@ -0,0 +1,52 @@
+package nextstep.qna.domain;
+
+import nextstep.qna.CannotDeleteException;
+import nextstep.users.domain.NsUser;
+
+import java.time.LocalDateTime;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.stream.Collectors;
+
+public class Answers {
+    private List<Answer> answers;
+
+    public Answers() {
+        this.answers  = new ArrayList<>();
+    }
+
+    public Answers(List<Answer> answers) {
+        this.answers = answers;
+    }
+
+    public void addOne(Answer answer) {
+        this.answers.add(answer);
+    }
+
+    public void addList(List<Answer> answers) {
+        this.answers.addAll(answers);
+    }
+
+    public List<Answer> getAnswers() {
+        return this.answers;
+    }
+
+    public boolean hasAnswers() {
+        return !this.answers.isEmpty();
+    }
+
+    public List<DeleteHistory> createDeleteHistory() {
+        return answers.stream().map(answer -> new DeleteHistory(ContentType.ANSWER,
```

Comment:
```suggestion
        return answers.stream()
        .map(answer -> new DeleteHistory(ContentType.ANSWER,
```
한줄의 점 하나로 가독성을 지켜봐요.